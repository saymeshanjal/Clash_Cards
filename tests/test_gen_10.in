17
12 35 2
4 19 5
4 4 2
4 18 2
16 29 3
12 0 2
12 44 3
16 3 3
10 11 2
4 19 5
8 38 3
12 23 4
2 46 3
6 37 3
8 8 2
16 22 4
16 29 5