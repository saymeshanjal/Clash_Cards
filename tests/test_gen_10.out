126 170 628
0 6 0
0 3 3
1 2 3
2123 10747 0
0 924 0
412 512 0
1287 11583 0
6 48 198
0 6 0
27 43 0
7 762 155
0 2 0
6 14 0
2 20 48
169 10765 1936
348 12522 0
